i18n/zh.yaml

```
@@ -0,0 +1,259 @@
+
+- id: translations-available
+  translation: "可用翻译"
+
+- id: read-in-language
+  translation: "阅读本文（%s 语言版）"
+
+- id: also-available-in
+  translation: "其他可用语言版本"
+
+- id: current-language
+  translation: "当前语言"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)